projects/wm-core/src/ugc-details/ugc-track-data/ugc-track-data.component.html

```diff
@@ -1,38 +1,38 @@
 <div class="wm-ugc-category-title ion-margin-top">{{"Dettagli" | wmtrans}}</div>
 <ion-list>
-  <ion-item class="wm-ugc-track-data-item">
+  <ion-item class="wm-ugc-track-data-item" *ngIf="(time$|async) && (time$|async) > 0">
```

Comment:
non si fa cosi tu qui fai due subscribes(un async è una subscribe) devi usare la direttiva ngIf su ogni ion item questo ti consente di fare un solo async e ti controlla se il valore è diverso da 0 in tanto o == false
```   
*ngIf="time$|async as time"
```